entries/m/migros.ch.json

```
@@ -1,17 +1,25 @@
 {
   "Migros": {
     "domain": "migros.ch",
-    "url": "https://www.migros.ch",
     "tfa": [
       "sms",
-      "totp"
+      "totp",
+      "custom-software"
```

```suggestion
      "custom-software",
      "u2f"
```

If you support FIDO2/U2F as a second-factor in addition to passkeys, the `u2f` element should be added. Do you have a documentation link for it?

---

Sorry no. We don't have more documentation (just [Migros Account Help](https://account.migros.ch/help) ). We belive that the term passkey should be suffisant. In our help page, we point out, that the password can still be used -> https://account.migros.ch/help#securePasskey